code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -128,24 +151,60 @@
 
 
 /obj/item/weapon/scalpel/laser
-	heat_production = 0
+	icon_state = "stabslash"
 	damtype = "fire"
 	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
 	var/cauterymode = 0 //1 = cautery enabled
+	var/obj/item/weapon/scalpel/laser/held
+
+/obj/item/weapon/scalpel/laser/New()
+	..()
+	icon_state = "scalpel"
+	held = new /obj/item/weapon/cautery/laser(src)
 
 /obj/item/weapon/scalpel/laser/attack_self(mob/user)
-	if(!cauterymode)
+	if(!cauterymode && held)
 		to_chat(user, "You disable the blade and switch to the scalpel's cautery tool.")
 		heat_production = 1600
 		sharpness = 0
 		sharpness_flags = 0
+	else if(!held)
+		to_chat(user, "\The [src] lacks a cautery attachment.")
+		return
 	else
 		to_chat(user, "You return the scalpel to cutting mode.")
 		heat_production = 0
 		sharpness = initial(sharpness)
```

Comment:
You have not given the laser scalpel a sharpness value in its type definition. It has no initial sharpness, nor do you appear to give it sharpness later. Did you intend to?

Reply:
It's a child of the scalpel so it get it from there.

Reply:
Also, is it intentional that the scalpel has 0 sharpness while in cautery mode despite the cautery itself having 1.5 sharpness?

Reply:
It's intended to be like that where it loses it's sharpness when in cautry mode. Issue resolved on the cautery by removing it's sharpness. Not sure if HOT_EDGE will cause an issue but if it needs to be removed so be it.